src/shared_modules/keystore/include/argsParser.hpp

```diff
@@ -0,0 +1,113 @@
+/*
+ * Wazuh keystore
+ * Copyright (C) 2015, Wazuh Inc.
+ * January 25, 2024.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _CMD_ARGS_PARSER_HPP_
+#define _CMD_ARGS_PARSER_HPP_
+
+#include <iostream>
+#include <string>
+
+/**
+ * @brief Class to parse command line arguments.
+ */
+class CmdLineArgs
+{
+public:
+    /**
+     * @brief Constructor for CmdLineArgs.
+     * @param argc Number of arguments.
+     * @param argv Arguments.
+     */
+    explicit CmdLineArgs(const int argc, const char* argv[])
+        : m_columnFamily {paramValueOf(argc, argv, "-f")}
+        , m_key {paramValueOf(argc, argv, "-k")}
+        , m_value {paramValueOf(argc, argv, "-v")}
+    {
```

Comment:
the -h option should be added.
currently if used it is considered an error
```shell
$ ./wazuh-keystore -h
Switch value: -f not found.

Usage: wazuh-keystore <option(s)>
Options:
        -h                      Show this help message
        -f COLUMN_FAMILY        Specifies the target column family for the insertion.
        -k KEY                  Specifies the key for the key-value pair.
        -v VALUE                Specifies the value associated with the key.

Example:
        ./wazuh-keystore -f indexer -k username -v admin
```

Reply:
Added a check at `paramValueOf()`